kDrive/UI/Controller/Files/File List/FileListViewController.swift

```diff
@@ -229,6 +229,7 @@ class FileListViewController: UIViewController, UICollectionViewDataSource, Swip
 
         navigationController?.setInfomaniakAppearanceNavigationBar()
 
+        guard viewModel != nil else { return }
```

Comment:
This is probably temporary. State restoration needs an overhaul

Reply:
Yep I agree. 
For the sake of the app's stability, I'm ok merging and using this in a beta, but IMO we need to check it more thoroughly before going live with this.